.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+	"name": "Azure Cosmos DB Explorer",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"onCreateCommand": ".devcontainer/oncreate",
```

Comment:
Weird that one of these is relative to current dir, and the other is to repo root.  I assume this works though?

Reply:
It does work, yeah. There's a reason for it, but it's definitely a little obscure. `build.dockerfile` specifies the value for the `-f` argument to `docker build`. So, this is effectively saying: `docker build -f Dockerfile`. In order to avoid the image pulling in the contents of the entire repo, that command is run **from the `.devcontainer`** directory.